vdb/recon/__init__.py

```diff
@@ -68,18 +68,28 @@ def reprargs(trace, fmt, args):
         r.append(rstr)
     return r
 
+def detect_cc(trace):
```

Comment:
i like this approach (calling `detect_cc`) much better than inline hand-jamming...

please define the dictionary outside the function, so it doesn't get recreated every call.

also, these are the defaults (although i386/linux should be cdecl), but not necessarily the reality.  what happens when the code gets it wrong?  what's worst-case scenario?

thanks!